src/main/kotlin/org/jetbrains/research/ml/ast/transformations/PyElementGeneratorExt.kt

```diff
@@ -0,0 +1,21 @@
+package org.jetbrains.research.ml.ast.transformations
```

Comment:
We have similar functions in the PyUtild object. I think we should refactor code and unify work with `PyElementGenerator`. Please, add todo about it and I will do it after merging this PR into the master branch.